<?xml version="1.0"?>
<launch>
  <arg name="output" default="screen"/>

  <!-- param file is all ros related params -->
  <!-- cfg file configures TLD -->
  <arg name="multi_cftld_param_file" default="$(find multi_cftld_ros)/config/default.yaml" />
  <arg name="multi_cftld_config_file" default="$(find multi_cftld_ros)/config/cftld.cfg" />

  <!-- temp -->
  <!--node name="republisher" type="republish" pkg="image_transport"
    args="compressed in:=bebop/image_raw raw out:=bebop/image_raw" output="$(arg output)" /-->

  <node pkg="multi_cftld_ros" type="multi_cftld_ros_node" name="multi_cftld_ros" output="$(arg output)" respawn="true">
    <remap from="image_raw" to="bebop/image_raw" />
    <remap from="init" to="multi_cftld/init" />
    <remap from="reset" to="multi_cftld/reset" />
    <remap from="tracks" to="multi_cftld/tracks" />
    <remap from="debug_image" to="cftld/debug_image_raw" />

    <param name="tld_config_file" value="$(arg multi_cftld_config_file)" />
    <rosparam command="load" file="$(arg multi_cftld_param_file)" />
  </node>
</launch>
